css/style.css

```diff
@@ -401,89 +401,83 @@ a:visited {
 /*================*/
 /*===Projects===*/
 /*===============*/
-/*
-.projectC {
-  display: flex;
-  position: relative;
-  width: 100%;
-  height: 400px;
-  background-color: none;
-}
-
-.projectC .main {
+.projectsC {
   display: flex;
-  position: absolute;
-  left: 0;
-  right: 0;
-  margin: auto;
-  top: 0;
-  bottom: 0;
+  flex-direction: column;
   align-items: center;
   justify-content: center;
   width: 100%;
-  height: 400px;
-  background-color: none;
+  margin-top: 60px;
+  padding: 50px 0;
+  background-color: #f9f9f9; /* Optional: Background color */
 }
 
-.projectC .main .textBig {
-  display: flex;
-  position: absolute;
-  left: 0;
-  right: 0;
-  margin: auto;
-  top: 90px;
-  width: 100%;
-  height: 230px;
-  align-items: start;
-  justify-content: center;
-  font-family: Montserrat;
-  font-weight: bold;
-  font-size: 58px;
-  background-color: none;
+.projectsC .title {
   text-align: center;
-  line-height: 1.2em;
+  font-family: Montserrat;
+  font-weight: 700;
+  font-size: 55px;
   color: #272341;
-  letter-spacing: 0.5px;
-  flex-wrap: wrap;
-  flex-direction: row;
 }
 
-.projectC .main .textSmall {
-  display: flex;
-  position: absolute;
-  left: 0;
-  right: 0;
-  margin: auto;
-  top: 200px;
-  width: 70%;
-  height: 230px;
-  align-items: start;
-  justify-content: center;
-  font-family: Poppins;
-  font-weight: 300;
-  font-size: 18px;
-  background-color: none;
+.projectsC .titleSmall {
   text-align: center;
-  line-height: 1.2em;
-  color: #96989b;
-  flex-wrap: wrap;
-  letter-spacing: 0.3px;
-  flex-direction: row;
+  font-family: Poppins;
+  font-weight: 400;
+  font-size: 20px;
+  color: #777;
+  margin-top: 10px;
 }
 
-.projectC .main .btnCont {
+.projectsC .container {
   display: flex;
-  position: absolute;
-  left: 0;
-  right: 0;
-  margin: auto;
-  bottom: 0;
+  flex-direction: column; /* Stack projects vertically */
   align-items: center;
   justify-content: center;
-  width: 100%;
-  height: 200px;
-  background-color: none;
-} */
+  margin-top: 20px;
+  gap: 20px; /* Space between project items */
+}
+
+.projectItem {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #ffffff;
+  border-radius: 10px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  padding: 20px;
+  width: 80%; /* Adjust width as needed */
+  transition: transform 0.3s; /* Smooth hover effect */
+}
+
+.projectItem:hover {
+  transform: scale(1.02); /* Slightly scale up on hover */
+}
+
+.projectIcon {
+  font-size: 40px; /* Icon size */
+  margin-right: 20px;
+}
+
+.projectText {
+  flex-grow: 1; /* Take remaining space */
+}
+
+.projectText .text {
+  font-weight: 500;
+  font-size: 24px;
+  color: #272341;
+}
+
+.projectText .textMain {
+  font-size: 16px;
+  color: #666;

```

Comment:
you should add font-family:"Montserrat" or font-family:"Poppins" and then select which looks the best